<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportFrame</class>
 <widget class="QFrame" name="ExportFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Shadow::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Export parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="OutputFormat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Convert to:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Postprocessing:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxUseForHttp">
        <property name="text">
         <string>Use profile for OPDS/Web server</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="CurrentTools">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>nothing</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="radioDevice">
       <property name="text">
        <string>Send to de&amp;vice</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioEmail">
       <property name="text">
        <string>Send &amp;to e-mail</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="PostprocessingCopy">
            <property name="text">
             <string>Use postprocessing for copy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="askPath">
            <property name="text">
             <string>Ask path before export</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_path">
            <property name="text">
             <string>Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Path"/>
          </item>
          <item>
           <widget class="QCheckBox" name="originalFileName">
            <property name="text">
             <string>Use original file name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_exportname">
            <property name="text">
             <string>File name:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="ExportFileName">
              <property name="toolTip">
               <string notr="true">&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;
&lt;p&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%a&lt;/span&gt; - автор книги. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%nl&lt;/span&gt; - фамилия. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%nm&lt;/span&gt; - отчество. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%nf&lt;/span&gt; - имя. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%li&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;%fi&lt;/span&gt;, &lt;span style=&quot; font-weight:700;&quot;&gt;%mi&lt;/span&gt; - первые буквы фамилии,имени,отчества. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%abbrs&lt;/span&gt; - аббревиатура серии.&lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%b&lt;/span&gt; - заголовок книги. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%s&lt;/span&gt; - серия. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%nx &lt;/span&gt;- номер в серии, где x-число знаков. &lt;br/&gt;
&lt;span style=&quot; font-weight:700;&quot;&gt;%id&lt;/span&gt; - id книги в библиотеке
&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="placeholderText">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="transliteration">
              <property name="text">
               <string>transliteration</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::FieldGrowthPolicy::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="Email">
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>e-mail</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Subject:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="mail_subject">
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string notr="true">freeLib</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Server:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,3">
              <item>
               <widget class="QLineEdit" name="Server">
                <property name="inputMask">
                 <string notr="true"/>
                </property>
                <property name="placeholderText">
                 <string>server address</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Type:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="ConnectionType">
                  <item>
                   <property name="text">
                    <string notr="true">Tcp</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">Ssl</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string notr="true">Tsl</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Port:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Port">
                  <property name="inputMask">
                   <string notr="true">00000</string>
                  </property>
                  <property name="placeholderText">
                   <string>port</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>User:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
              <item>
               <widget class="QLineEdit" name="User">
                <property name="inputMask">
                 <string notr="true"/>
                </property>
                <property name="placeholderText">
                 <string>user name</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Password:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="Password">
                  <property name="inputMask">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::EchoMode::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>password</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Pause:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QSpinBox" name="PauseMail"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="from_email">
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>e-mail</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>OutputFormat</tabstop>
  <tabstop>CurrentTools</tabstop>
  <tabstop>radioDevice</tabstop>
  <tabstop>radioEmail</tabstop>
  <tabstop>ExportFileName</tabstop>
  <tabstop>transliteration</tabstop>
  <tabstop>from_email</tabstop>
  <tabstop>Email</tabstop>
  <tabstop>mail_subject</tabstop>
  <tabstop>Server</tabstop>
  <tabstop>ConnectionType</tabstop>
  <tabstop>Port</tabstop>
  <tabstop>User</tabstop>
  <tabstop>Password</tabstop>
  <tabstop>PauseMail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
